web/src/components/atoms/ContentSection/index.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from "@emotion/styled";
+import { ReactNode } from "react";
+
+export type Props = {
+  title: string;
+  danger?: boolean;
+  extra?: ReactNode;
+  children?: ReactNode;
+};
+
+const ContentSection: React.FC<Props> = ({ title, extra, children, danger }) => {
+  return (
+    <Wrapper danger={danger}>
+      <Header>
+        <Subtitle>{title}</Subtitle>
```

Comment:
In the context of this component, this is a Title, not Subtitle

Reply:
@KaWaite , you are right.
Fixed.